package.json

```diff
@@ -247,8 +247,11 @@
     "pretty": "tslint --fix './src/**/*.ts' './test/**/*.ts'",
     "pre-commit": "lint-staged",
     "test": "npm run compile && node ./out/test/runTest.js",
+    "test-headless": "xvfb-run -a npm run test",
     "test-unit": "cross-env MOCHA_GREP=@unit npm test --silent",
+    "test-unit-headless": "xvfb-run -a npm run test-unit",
     "test-integration": "cross-env MOCHA_GREP=@integration npm test --silent",
+    "test-integration-headless": "xvfb-run -a npm run test-integration",
```

Comment:
[F[ These headless options means that the tests can run from the command line in the devcontainer and in the CI via for linux via the same command.

Reply:
excellent this makes a lot of sense especially what we saw in the previous PR where we discussed this and the documentation provided my microsoft.
https://code.visualstudio.com/api/working-with-extensions/continuous-integration#github-actions